manifests/agent/php.pp

```diff
@@ -61,6 +61,7 @@
   $newrelic_ini_transaction_tracer_stack_trace_threshold = undef,
   $newrelic_ini_transaction_tracer_threshold             = undef,
   $newrelic_ini_webtransaction_name_files                = undef,
+  $newrelic_php_extension_enabled                        = true,
```

Comment:
Minor nitpick: the extension line is about loading an extension, I think the naming could be confusing with the newrelic enabled option.

I'd use a different name, such as `$newrelic_ini_load_php_exentesion` for this variable

Reply:
fixed